### DISCUSSION TITLE
vue-cesium本地引入cesium.js文件报错的问题

### DISCUSSION THREAD

**Discussion by songwei0814** [zouyaoji/vue-cesium#233] [Q&A]
在main.js文件中引入cesium.js(unpkg下载)，控制台报错，node_modules/vue-cesium/lib/viewer/index.js文件报错

**Comment by zouyaoji** [comment]
看一下模板项目。

https://zouyaoji.top/vue-cesium/#/zh-CN/component/quickstart#shi-yong-mo-ban
